File: scratch/inspect_registration.js
const fs = require('fs');
const content = fs.readFileSync('c:/Users/MI/OneDrive/Desktop/koduvellisahitholsv/registration.html', 'utf8');
const lines = content.split('\n');

lines.forEach((line, i) => {
    if (line.includes('pending') || line.includes('queue') || line.includes('list')) {
        if (line.includes('function') || line.includes('push') || line.includes('localStorage') || line.includes('register')) {
            console.log(`${i+1}: ${line.trim()}`);
        }
    }
});

File: scratch/inspect_registration.py
with open('c:/Users/MI/OneDrive/Desktop/koduvellisahitholsv/registration.html', 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if 'pending' in line or 'queue' in line or 'list' in line:
        if 'function' in line or 'push' in line or 'localStorage' in line:
            print(f"{i+1}: {line.strip()}")
